### DISCUSSION TITLE
Know the name or number of the current tab

### DISCUSSION THREAD

**Discussion by ravikantc17** [manojadams/mui-forms#73] [Q&A] [answered]
<img width="897" alt="Screenshot 2024-05-08 at 2 24 57 PM" src="https://github.com/manojadams/mui-forms/assets/115449797/f152aa65-8b6e-407d-b5f9-8dcea59d976f">

Hi, In the above screen I want to know on which tab we are currently at. If it reaches on Health Condition, we want to refresh the form and land at the Health Condition.
thanks

**Comment by venkateshpullaganti** [comment]
@manojadams 

Is there a way that we can get the current tab name while onNext or onSubmit ?

**Reply by manojadams** [reply, marked as answer]
@venkateshpullaganti 
onNext contains a second parameter which is the page number of the page being submitted.
